### PR TITLE
fix: small fixes to saving for release review

### PR DESCRIPTION
**Pull request by davidmenendez**

### CODE REVIEW

File: packages/cloud-cognitive/src/components/Saving/Saving.js [hunk 1]
```diff
@@ -59,24 +59,21 @@ export let Saving = ({
...
   const statusObj = getStatusObj();
-
-  const classnames = cx(`${pkg.prefix}-saving`, {
-    [className]: className,
-  });
+  const blockClass = `${pkg.prefix}--saving`;
 
   return (
-    <div className={classnames}>
+    <div className={cx(blockClass, className)}>
```
Comment (lee-chase) [inline packages/cloud-cognitive/src/components/Saving/Saving.js:65]: Format of selector should be `prefix--Block__Element--Modifier`

**Review by lee-chase — CHANGES_REQUESTED** [review]
See comment otherwise ok

File: packages/cloud-cognitive/src/components/Saving/Saving.js (hunk 1 above)
Comment (dcwarwick) [reply packages/cloud-cognitive/src/components/Saving/Saving.js:65]: _i.e._ `${blockClass}__message`, `${blockClass}__error-icon`, `${blockClass}__text`, `${blockClass}__buttons`.

Comment (davidmenendez) [reply packages/cloud-cognitive/src/components/Saving/Saving.js:65]: gotcha

**Comment by davidmenendez** [comment]
changes made

**Review by lee-chase — APPROVED** [review]

**Pull request merged**